upstairs/src/lib.rs

```diff
@@ -4918,6 +5649,9 @@ impl Guest {
     pub fn deactivate(&self) -> Result<BlockReqWaiter, CrucibleError> {
         // Disable any more IO from this guest and deactivate the downstairs.
         // We can't deactivate if we are not yet active.
+        //
+        // ZZZ This should come from the guest, not ourselves.
```

Comment:
XXX?

Reply:
ZZZ is my "come back to this and deal with before I make a PR".
Clearly I have failed.  Will remove.

Reply:
Removed in 9acce4f